cfme/fixtures/vm.py

```diff
@@ -100,20 +100,16 @@ def _get_vm_name(request):
 
 @pytest.fixture(scope="module")
 def create_vm_modscope(setup_provider_modscope, request, provider):
-    if request.param:
-        template_type = request.param
-    else:
-        pytest.error('Any appropriate Template was not passed to the fixture.')
+    # If no template parameter is passed then use DSL.
+    template_type = getattr(request, 'param', 'damnSmallLinux')
```

Comment:
Thanks for the PR Satya. This was a topic that we had discussed here: https://github.com/ManageIQ/integration_tests/pull/9186#discussion_r316870496 briefly. 
Do you still think we need the default? Not having a default was a decision we made to force contributors to make a concious choice of templates.

Reply:
I have went through the discussion and I got to know the purpose. But if you look into the changes I have made there are not conflicting with the actual implementation.
We also have many tests which only requires a basic template(DSL), so in that case we have to pass/parameterize each tests which I think is not that easy. :smile: 
What's say .?

Reply:
You have a good point too. I'm okay going ahead if we have reviewers consensus.

Reply:
I wouldn't agree with trying to pick a specific template name like this, hardcoded.

If anything, it should default to looking at the provider yaml for a small_template definition, and using that as the default when one isn't parametrized.

Reply:
Moving to WIPTEST per ^ comment.

Reply:
@mshriver I have added `small_template` in place of DSL.